frontend/src/lib-components/atoms/buttons/ReturnButton.tsx

```diff
@@ -25,8 +24,8 @@ export const ReturnButtonWrapper = styled.div<WrapperProps>`
     justify-content: flex-start;
   }
 
-  @media (max-width: ${tabletMin}) {
-    margin-left: ${defaultMargins.m};
+  @media (max-width: 1215px) {
```

Comment:
Where does this magic number come from?

Reply:
`lib-components/layout/Container.tsx:Container`